sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -353,7 +365,26 @@ struct graphs_support;
 } // namespace device
 } // namespace info
 
-class node {};
+class dynamic_parameter {
+public:
+  template <typename T>
+  explicit dynamic_parameter(T param, std::string label = "");
```

Comment:
We should update this API would work with backend interop, e.g if a kernel uses `set_args`/set_arg` How can we tie that to a dynamic parameter that can be update. This would also enable scalar arguments to be updated.

Reply:
Yes, it can be used with kernel bundles and explicit set_arg(s) apis